src/components/number-picker/number-picker.component.spec.ts

```diff
@@ -3,15 +3,18 @@ import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { FormBuilder, FormGroup, FormsModule, ReactiveFormsModule, Validators } from '@angular/forms';
 import { NumberPickerModule } from './number-picker.module';
 
+/** Basic number picker example */
```

Comment:
Don't like this description. Each test fixture has a specific purpose; this one tests the number picker in a reactive forms context.

Try not to add comments unless they add real value.